ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -916,11 +916,16 @@ def _gather_additional_module_files(self, module_path: str) -> Dict[str, str]:
 
                 if not should_ignore:
                     for serialized_pattern in self._serialized_files:
-                        if fnmatch.fnmatch(relative_path, serialized_pattern) or fnmatch.fnmatch(
-                            filename, serialized_pattern
-                        ):
-                            should_ignore = True
-                            break
+                        if serialized_pattern in ["__init__.py", "workflow.py", "inputs.py", "state.py"]:
```

Comment:
we shouldn't need to do any hardcoding here at all. we should look into using `glob.glob` instead so that it matches against all filename specification's in `_serialized_files` exactly, unless there are asterisks